content/articles/road-layout-design-programatically-using-matlab/index.md

```diff
@@ -0,0 +1,177 @@
+
+### Road layout design programmatacally using matlab
+### Table of content
+- [Road layout design programmatacally using matlab](#road-layout-design-programmatacally-using-matlab)
+- [Table of content](#table-of-content)
+- [Introduction](#introduction)
+- [### Prerequisites](#-prerequisites)
+- [Straight road](#straight-road)
+- [Roads with lanes](#roads-with-lanes)
+- [Road intersection](#road-intersection)
+- [Curved roads layout](#curved-roads-layout)
+- [Roundabout road layout](#roundabout-road-layout)
+- [Elevated road layout](#elevated-road-layout)
+- [Overpass roads](#overpass-roads)
+- [Conclusion](#conclusion)
+
+
+
+### Introduction
+In designing road layout, we use the `drivingscenario` function. `scenario` function allows for simulation of an activity or a structure that can be found in the real world. `Drivingscenario` function can be used to create different designs of road layout by plotting the road layout in 2D or 3d graphs. This is done by specifying the coordinates of the road center using function `roadCentre=[];` and the road width then plotting these points on the graph.
+
+This article will discuss the design of the  following road layout using Matlab functions:
+
+- Straight road.
+- Roads with lanes.
+- Intersecting roads.
+- Curved road layouts.
+- Roundabout layout.
+- Elevated road layout.
+- Overpass road layout.
+
```

Comment:
Restructure the article, such that we've got this intro part appearing on the top! followed by the ToC in that order.